src/app.js

```diff
@@ -101,6 +106,7 @@ app.use('/index', userOrBackendAuthorization, indexRouter);
 app.use('/fhir', userOrBackendAuthorization, fhirRouter);
```

Comment:
I was thinking about that, I think the auth would fail but I'm not sure.  If we put it in the databaseReady function it wouldn't be set up before then, though, so we could do that as well.

Reply:
Hmmm. It's less than ideal to have both the DB config and still use environment variables. But if it's a big mess in the tests then I guess it's ok

Reply:
I mean, it's nice to have everything in one place. In the description of the task we did say some things are not meant to change, or at least not without a server restart, for example the server base URL. So if it's a lot easier to keep something as an env var, ok.